tests/playbooks/kind_crud.yml

```diff
@@ -0,0 +1,735 @@
+---
+- name: CRUD test for VM
+  hosts: localhost
+  connection: local
+  tasks:
+    - name: Create test-fedora VM
+      kubevirt_raw:
+        state: present
+        name: test-fedora
+        namespace: default
+        inline:
+          apiVersion: kubevirt.io/v1alpha2
+          kind: VirtualMachine
+          metadata:
+            name: test-fedora
+            namespace: default
+            labels:
+              kubevirt.io/vm: test
+          spec:
+            running: true
+            template:
+              metadata:
+                labels:
+                  kubevirt.io/vm: test
+              spec:
+                domain:
+                  resources:
+                    requests:
+                      memory: 512M
+                  devices:
+                    disks:
+                    - volumeName: myvolume
+                      name: mydisk
+                      disk:
+                        bus: virtio
+                    - name: cloudinitdisk
+                      volumeName: cloudinitvolume
+                      cdrom:
+                        bus: virtio
+                volumes:
+                - name: myvolume
+                  registryDisk:
+                    image: kubevirt/fedora-cloud-registry-disk-demo:latest
+                - cloudInitNoCloud:
+                    userData: |
+                      #cloud-config
+                      hostname: test-fedora
+                      users:
+                        - name: kubevirt
+                          gecos: KubeVirt Project
+                          sudo: ALL=(ALL) NOPASSWD:ALL
+                          passwd: $6$JXbc3063IJir.e5h$ypMlYScNMlUtvQ8Il1ldZi/mat7wXTiRioGx6TQmJjTVMandKqr.jJfe99.QckyfH/JJ.OdvLb5/OrCa8ftLr.
+                          shell: /bin/bash
+                          home: /home/kubevirt
+                          lock_passwd: false
+                          ssh_authorized_keys:
+                            - ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC+KMp1F4SrKitvgILf8Q2GQbcOslIg4sfqUOf+m832zwoS2Z2ndmpr/3F+NtLq56cTaEzK9IPBIkNLJfsPqRsToLBYZD+l4S5FfuZ2xwCxGBH+TVXApO+SiD6c84rmjOx47665iQvMHKL+n/5gVvSdYDuegNKnj4rRr/eHnG2yC4TVZl3oHI7TPOUJT+kKjSWP1UesWTZmmck39IaFSmorg31X7g9hJHwq9JEDQilcbnIyqDZKiH6Ju4GjOU8mqhazBFB4qu/QDDb25pDpPd2pQGBilGvm7gwJCVnyDk9YZQU7gYE734KLDf5tCKMmEQSjwFx2Tj9mfZveCIJkaj3T kubevirt
+                  name: cloudinitvolume
+      register: vm_facts
+  
+    - name: Fail if VM create task was not 'changed'
+      assert:
+        that:
+        - "vm_facts is changed"
+        fail_msg: "Create VM task did not return changed"
+    
+    - name: wait for changes to appear
+      wait_for: timeout=15
```

Comment:
Timeout is with seconds. Can we wait on object to change like we had in import vm apb:

> - Name: Check PVC bound status
>  command: "oc get pvc {{ vm_name }}-disk-01 -o jsonpath='{ {{pvc_status_path}} }'"
>  register: pvc_bound_status
>  until: pvc_bound_status.stdout == 'Bound'
>  retries: 60
>  delay: 3